landingai/data_management/media.py

```diff
@@ -16,13 +16,15 @@
 import PIL.Image
 from PIL.Image import Image
 from tqdm import tqdm
+import nest_asyncio
```

Comment:
This library is needed to run asynchronous code in Jupyter notebooks. See the "[Introduction](https://pypi.org/project/nest-asyncio/)" section of this library.

Reply:
Ideally, we should stop using asyncio internally and start using `requests`. Externally speaking, the whole library is non-asyncio (meaning that end users don't do `await out_function_call()` anywhere. It doesn't make much sense to use asyncio internally, even more because we are dealing with the event loop ourselves, and blocking the call anyway.

Do you think it would take us too much time to refactor the `data_management` module to use `requests` library instead of adding yet another dependency for asyncio?

Reply:
Done